src/syrupy/__init__.py

```diff
@@ -63,6 +64,11 @@ def pytest_sessionstart(session: Any) -> None:
         warn_unused_snapshots=config.option.warn_unused_snapshots,
         update_snapshots=config.option.update_snapshots,
         base_dir=config.rootdir,
+        targeted_items={
+            arg
+            for arg in config.invocation_params.args
```

Comment:
(nb) worth checking if pytest exposes these "files" in some nicer API rather than looping through invocation args. I couldn't find anything when I previously looked

Reply:
We now only use `invocation_params` to determine whether we are running `pytest` with paths or not. We rely on the `ran_items` list compiled downstream to determine the actual paths. More robust that way. I still don't know a good whether there's another way to determine the `pytest` run mode.